pkg/core/block.go

```diff
@@ -46,6 +46,10 @@ func (b *Block) rebuildMerkleRoot() error {
 
 // Verify the integrity of the block.
 func (b *Block) Verify(full bool) bool {
+	// there has to be some transaction inside
```

Comment:
nit: add dot in the end

Reply:
I'm not sure it's applicable for these internal comments. They're not visible via godoc.

Reply:
haven't found anything specific to that but:
https://golang.org/doc/effective_go.html#commentary
and will be nice to have consistent style

Reply:
Well, internal function comments can be quite varying, like here:
https://github.com/golang/go/blob/master/src/sort/sort.go
https://github.com/golang/go/blob/master/src/mime/mediatype.go

But still there probably is a tendency to follow the same rules for them.